src/components/TitlesCarousel/TitlesCarousel.scss

```diff
@@ -0,0 +1,48 @@
+.wrapper {
+    width: 100%;
+    position: relative;
+    display: flex;
+    .carousel__prev, .carousel__next {
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    outline: 0;
+    cursor: pointer;
+    position: absolute;
+    }
+    .carousel__prev {
```

Comment:
I guess you can move these styles to the .carousel{} and use the &__prev syntax

Reply:
prev and next are in a wrapper, not a carousel so I can change class

Reply:
done